cmd/virt-handler/virt-handler.go

```diff
@@ -189,23 +197,37 @@ func (app *virtHandlerApp) Run() {
 	ws.Route(ws.GET("/api/v1/namespaces/{namespace}/virtualmachines/{name}/console").To(console.Console))
 	ws.Route(ws.GET("/api/v1/namespaces/{namespace}/virtualmachines/{name}/migrationHostInfo").To(migrationHostInfo.MigrationHostInfo))
 	restful.DefaultContainer.Add(ws)
-	server := &http.Server{Addr: app.Service.Address(), Handler: restful.DefaultContainer}
+	server := &http.Server{Addr: app.Address(), Handler: restful.DefaultContainer}
 	server.ListenAndServe()
 }
 
+func (app *virtHandlerApp) AddFlags() {
+	app.InitFlags()
+
+	app.BindAddress = defaultHost
+	app.Port = defaultPort
+	app.LibvirtUri = libvirtUri
+
+	app.AddCommonFlags()
+	app.AddLibvirtFlags()
+
+	flag.StringVar(&app.HostOverride, "hostname-override", hostOverride,
+		"Kubernetes pod to monitor for changes")
```

Comment:
The description should be more something like: "Name under which the node is registered in kubernetes, where this virt-handler instance is running on".